sdk/samples/iot/README.md

```diff
@@ -176,13 +173,53 @@ To run the samples, ensure you have the following programs and tools installed o
     git clone https://github.com/Azure/azure-sdk-for-c.git
     ```
 
+- If running a DPS sample: `paho_iot_provisioning_sample`, `paho_iot_provisioning_sas_sample`
+  - Have an [Azure IoT Hub Device Provisioning Service (DPS)](https://docs.microsoft.com/azure/iot-dps/quick-setup-auto-provision) created.
+
+- If running a Plug and Play sample: `paho_iot_hub_pnp_sample`, `paho_iot_hub_pnp_component_sample`
+  - Have the most recent version of [Azure IoT Explorer](https://github.com/Azure/azure-iot-explorer/releases) installed and connected to your Azure IoT Hub. More instructions on can be found [here](https://docs.microsoft.com/azure/iot-pnp/howto-use-iot-explorer).
+
+- If running a CBOR sample: `paho_iot_hub_twin_cbor_sample`
+  - Have the [MIT licensed](https://github.com/intel/tinycbor/blob/master/LICENSE) intel/tinycbor library installed.
+    <details><summary><i>Instructions:</i></summary>
+    <p>
+
+    Linux:
+
+    ```
+    git clone https://github.com/intel/tinycbor.git
+    cd tinycbor
+    git checkout v0.5.3
+    make
+    sudo make install
+    ```
+
+    - Please check your `/usr/local/lib` directory and update permissions on the installed libraries if needed.
+
+    Windows (Select the Command Prompt):
+
+    - x86 system: Open the Developer Command Prompt for Visual Studio.
+    - x64 system: Open the x64 Native Tools Command Prompt for Visual Studio.
+
+    ```
+    git clone https://github.com/intel/tinycbor.git
+    cd tinycbor
+    git checkout v0.5.3
+    NMAKE /F Makefile.nmake
+    ```
+
+    - Update your Path environment variable to include the tinycbor directory.
```

Comment:
Why is this required?

It might be helpful to add more details on what exactly I need to set my path environment var too, with an example.

Reply:
The CMake needs to know where to look for the tinycbor library.  Since the we don't know where the user will install it, I can tell CMake to search in a specific path (e.g., Env Path).  The user just has to include the library in the Env Path.

Reply:
The Linux install is nice, because `make install` puts the library in the /usr/local/lib.  That is one of the locations CMake automatically looks, so no extra instructions need to give to it.

Reply:
> I assumed if someone was running this on windows, they would know how to do it, or could look it up. Maybe that's a bad assumption.

On Windows, what is the default install location that I need to set my PATH env to once I call `NMAKE /F Makefile.nmake`? 
It would be helpful to use that as an example `PATH` so I, as the reader, don't have to spend time figuring this out :)

If I want to override that location to some other directory, how can I do that?

Reply:
Agreed. That makes a lot of sense.

> Otherwise, I think a basic "open the system properties window to edit the PATH environment variable" should suffice.

Yep. A tweak like this might just be good enough:
```suggestion
    - Open the system properties window and update your Path environment variable to include the location of the tinycbor directory you cloned above.
```